examples/configuration.nix

```diff
@@ -150,6 +161,28 @@
   # public access to the web interface.
   # nix-bitcoin.onionServices.btcpayserver.enable = true;
 
+  ### MEMPOOL
+  # Set this to enable mempool, a fully featured Bitcoin visualizer, explorer,
+  # and API service.
+  #
+  # services.mempool.enable = true;
+  #
+  # Possible options for the Electrum backend server:
+  #
+  # - electrs:
+  #   Small database size, slow when querying new addresses.
+  #   Enable with:
+  #     services.mempool.electrumServer = "electrs";
```

Comment:
Isn't this already enabled by default? Why list it here explicitly? This gives the user the idea that he has to explicitly enable it.